scripts/import-to-bn.py

```diff
@@ -0,0 +1,111 @@
+"""
+Binary Ninja plugin that imports a capa report,
+produced via `capa --json /path/to/sample`,
+into the current database.
+
+It will mark up functions with their capa matches, like:
+
+    ; capa: print debug messages (host-interaction/log/debug/write-event)
+    ; capa: delete service (host-interaction/service/delete)
+    ; Attributes: bp-based frame
+
+    public UninstallService
+    UninstallService proc near
+    ...
+
+To use, invoke from the Binary Ninja Tools menu, or from the 
+command-palette.
+
+This script will verify that the report matches the workspace.
+Check the log window for any errors, and/or the summary of changes.
+

```

Comment:
would you like to add your handle here for recognition?

Reply:
Sure, thanks. Will amend.